public/content/translations/ja/developers/docs/mev/index.md

```diff
@@ -24,7 +24,7 @@ lang: ja
 
 ### ガス・ゴルフ {#mev-extraction-gas-golfing}
 
-上記の力学に基づき、「ガス・ゴルフ（gas golfing）」（ガス代が最も安価になるようにトランザクションをプログラミングすること）の能力が競合上の優位性として確立されています。ガス・ゴルフにより、サーチャーはより高価なガス代を設定しつつ、ガス代の合計額を一定に抑えられるためです（ガス代＝ガス価格÷使用ガス量）。
+上記の力学に基づき、「ガス・ゴルフ（gas golfing）」（ガス代が最も安価になるようにトランザクションをプログラミングすること）の能力が競合上の優位性として確立されています。ガス・ゴルフにより、サーチャーはより高価なガス代を設定しつつ、ガス代の合計額を一定に抑えられるためです（ガス代＝ガス価格*使用ガス量）。
```

Comment:
Oops, not sure what happened to my comment before, but checked Crowdin, and this has been updated to use the `×` character... those updates will eventually go live, but still happy to patch this here in the meantime. We should use that character instead of the `*` though to match (we use Crowdin to manage all non-English content translations and updates)

```suggestion
上記の力学に基づき、「ガス・ゴルフ（gas golfing）」（ガス代が最も安価になるようにトランザクションをプログラミングすること）の能力が競合上の優位性として確立されています。ガス・ゴルフにより、サーチャーはより高価なガス代を設定しつつ、ガス代の合計額を一定に抑えられるためです（ガス代＝ガス価格×使用ガス量）。
```